build/root/usr/local/bin/ci_entrypoint_gpu-operator.sh

```diff
@@ -58,17 +58,35 @@ test_master_branch() {
 }
 
 test_commit() {
-    CI_IMAGE_GPU_COMMIT_CI_REPO="${1:-https://github.com/NVIDIA/gpu-operator.git}"
-    CI_IMAGE_GPU_COMMIT_CI_REF="${2:-master}"
-
+    gpu_operator_git_repo="${1:-}"
+    gpu_operator_git_ref="${2:-}"
     CI_IMAGE_GPU_COMMIT_CI_IMAGE_UID="ci-image"
 
-    echo "Using Git repository ${CI_IMAGE_GPU_COMMIT_CI_REPO} with ref ${CI_IMAGE_GPU_COMMIT_CI_REF}"
+    if [[ -z "$gpu_operator_git_repo" || -z "$gpu_operator_git_ref" ]]; then
+        echo "FATAL: test_commit must receive a git repo/ref to be tested."
+        return 1
+    fi
+
+    echo "Using Git repository ${gpu_operator_git_repo} with ref ${gpu_operator_git_ref}"
 
     prepare_cluster_for_gpu_operator
-    toolbox/gpu-operator/deploy_from_commit.sh "${CI_IMAGE_GPU_COMMIT_CI_REPO}" \
-                                               "${CI_IMAGE_GPU_COMMIT_CI_REF}" \
+
+    if [ -z "${GPU_OPERATOR_QUAY_BUNDLE_PUSH_SECRET:-}" ]; then
+        GPU_OPERATOR_QUAY_BUNDLE_PUSH_SECRET=/var/run/psap-entitlement-secret/openshift-psap-openshift-ci-secret.yml
+    fi
```

Comment:
```suggestion
    GPU_OPERATOR_QUAY_BUNDLE_PUSH_SECRET=${GPU_OPERATOR_QUAY_BUNDLE_PUSH_SECRET:-"/var/run/psap-entitlement-secret/openshift-psap-openshift-ci-secret.yml"}
```

Or could also just remove the `:-` in the conditional if you prefer the if then statement

Reply:
thanks, applied